pom.xml

```diff
@@ -252,7 +252,7 @@
         <gravitee-expression-language.version>1.9.2</gravitee-expression-language.version>
         <gravitee-reporter-api.version>1.22.0</gravitee-reporter-api.version>
         <gravitee-tracing-api.version>1.0.0</gravitee-tracing-api.version>
-        <gravitee-kubernetes-client.version>0.3.0</gravitee-kubernetes-client.version>
+        <gravitee-kubernetes-client.version>0.4.1</gravitee-kubernetes-client.version>
```

Comment:
@jhaeyaert can I merge this PR?

Reply:
Sorry, I can't say without more information. I can see that you're moving from a 0.3.x to a 0.4.x. Is there any potential breaking change or is it ok?

Reply:
No, there is not breaking change. It was just the beginning of this semantic release and the version jumped to 0.4.x